src/components/Amalgamation/BusinessTable.vue

```diff
@@ -0,0 +1,188 @@
+          :key="item.businessId"
+        >
+          <td class="business-name">
+            <strong>{{ item.name }}</strong><br>{{ item.email }}
+          </td>
+
+          <td class="business-type">
+            {{ GetCorpFullDescription(item.type) }}
+          </td>
+
+          <td class="business-address">
+            <BaseAddress
+              v-if="item.address"
+              :address="item.address"
+            />
+            <span v-else-if="item.jurisdiction">{{ item.jurisdiction }}</span>
+            <span v-else>Affiliate to view</span>
+          </td>
+
+          <td class="business-role">
+            {{ item.role }}
+          </td>
+
+          <td class="business-status">
+            <BusinessStatus :status="item.status" />
+          </td>
+
+          <td class="business-actions">
+            <v-btn
+              text
+              color="primary"
+              @click="removeBusiness(index)"
+            >
+              <v-icon
+                small
+                color="primary"
+              >
+                mdi-delete
+              </v-icon>
+              <span class="ml-2">Remove</span>
+            </v-btn>
+          </td>
+        </tr>
+      </tbody>
+    </template>
+  </v-simple-table>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Vue, Watch } from 'vue-property-decorator'
+import { Getter, Action } from 'pinia-class'
+import { useStore } from '@/store/store'
+import { BusinessStatuses } from '@/enums'
+import { DefineCompanyIF } from '@/interfaces'
+import { BaseAddress } from '@bcrs-shared-components/base-address'
+import BusinessStatus from './BusinessStatus.vue'
+import { GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module'
+
+@Component({
+  components: {
+    BaseAddress,
+    BusinessStatus
+  }
+})
+export default class BusinessTable extends Vue {
+  readonly GetCorpFullDescription = GetCorpFullDescription
+
+  @Getter(useStore) getAmalgamatingBusinesses!: any[]
+  @Getter(useStore) getDefineCompanyStep!: DefineCompanyIF
+  @Getter(useStore) getNameTranslationsValid!: boolean
+  @Getter(useStore) getShowErrors!: boolean
+  @Getter(useStore) isBaseCompany!: boolean
+  @Getter(useStore) isRoleStaff!: boolean
+
+  @Action(useStore) setAmalgamatingBusinesses!: (x: any[]) => void
+  @Action(useStore) setDefineCompanyStepValidity!: (x: boolean) => void
+  @Action(useStore) setIgnoreChanges!: (x: boolean) => void
+
+  get businesses (): any[] {
+    // *** TODO: use "map" to compute status from other info
+    //           eg, if business.goodStanding != true then status = ERROR_NIGS
+    return this.getAmalgamatingBusinesses.map(business => {
+      if (!business.goodStanding) business.status = BusinessStatuses.ERROR_NIGS
+      return business
+    })
+  }
+
+  removeBusiness (index: number): void {
+    const temp = this.getAmalgamatingBusinesses
+    temp.splice(index, 1)
+    this.setAmalgamatingBusinesses(temp)
```

Comment:
This is to set a new array in the store.

I had this working without using `temp` -- in other words, directly using the object that getAmalgamatingBusinesses returned -- but I think it's better to trigger the actions instead of changing the data directly.